db/models.py

```diff
@@ -19,34 +29,78 @@ def __str__(self) -> str:
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
-    actors = models.ManyToManyField(to=Actor, related_name="movies")
-    genres = models.ManyToManyField(to=Genre, related_name="movies")
+    genres = models.ManyToManyField(Genre)
+    actors = models.ManyToManyField(Actor)
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(null=True)
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
 
     def __str__(self) -> str:
-        return self.title
+        return str(self.created_at)
+
+    class Meta:
```

Comment:
Code Style: The `class Meta` is defined after all attributes. This is a good practice. Keep it up!